recipes/darwin-toolchain/all/conanfile.py

```diff
@@ -0,0 +1,117 @@
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+
+import os
+import platform
+import copy
+
+
+class DarwinToolchainConan(ConanFile):
+    name = "darwin-toolchain"
+    version = "1.0.9"
```

Comment:
Why hardcoding the version in the recipe instead of using the traditional approach?

Reply:
As mentioned quite unclearly in https://github.com/conan-io/conan-center-index/pull/5545#issuecomment-842308294, this is an arbitrary version that I used to bump everytime I made a change to the recipe, but we can decide of a different version scheme

Reply:
@theodelrieu if the version represents the recipe, then you should not put the files in `recipes/darwin-toolchain/all`, but in `recipes/darwin-toolchain/all/1.0.9`, then you can remove the version attribute here, and remove `recipes/darwin-toolchain/config.yml` (it is useless when the version number is the same as the subfolder name